mephisto/abstractions/blueprints/parlai_chat/parlai_chat_task_runner.py

```diff
@@ -199,6 +200,13 @@ def run_onboarding(self, agent: "OnboardingAgent") -> None:
                     },
                 )
             )
+        # Mark the agent as done, then wait for the incoming submit action
+        agent.mark_done()
+        while not agent.has_action.is_set():
+            done_act = agent.act()
+            if done_act is not None:
+                break
+            time.sleep(0.3)
```

Comment:
question: could we have this as a variable in case we wanted to find the optimal value?